src/types/SamsungPayType.res

```diff
@@ -0,0 +1,83 @@
+open Utils
+type payment3DS = {
+  \"type": string,
+  version: string,
+  data: string,
+}
+
+type paymentShippingAddress = {
+  shipping: Js.Json.t, // Assuming this is a complex object, we use Js.Json.t for now
+  email: string,
+}
+
+type paymentCredential = {
+  \"3_d_s": payment3DS,
+  card_brand: string,
+  // payment_currency_type: string,
+  // payment_last4_dpan: string,
+  // payment_last4_fpan: string,
+  card_last4digits: string,
+  // merchant_ref: string,
+  method: string,
+  recurring_payment: bool,
+  // payment_shipping_address: paymentShippingAddress,
+  // payment_shipping_method: string,
+}
```

Comment:
How do we handle billing address requirements for a processor?

Reply:
We are not collecting it via Samsung Pay, as it will add extra friction of filling all these details in Samsung Pay Sheet.  We will either accept that in create or confirm ( via HS SDK)